apis/keda/v1alpha1/scaledobject_types.go

```diff
@@ -184,6 +186,6 @@ func init() {
 }
 
 // GenerateIdentifier returns identifier for the object in for "kind.namespace.name"
-func (s *ScaledObject) GenerateIdentifier() string {
-	return GenerateIdentifier("ScaledObject", s.Namespace, s.Name)
+func (so *ScaledObject) GenerateIdentifier() string {
```

Comment:
This change is required here (or in the new code), because the linter/rules requires that all the functions from the same struct has the same name for the item, if I name it here with `s`, I need to use `s` for every function. I can revert his change here and update the code in the other places to use `s` instead of `so` but I felt `so` more accurate with the element we are managing. 
I'm happy with both so as you prefer

Reply:
okay, let's keep it this way :)